### PR TITLE
docs: Update documentation

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: docs/RESOURCES.md [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
+- Eric Lippert's [Fabulous adventures in coding](https://ericlippert.com/) is a brilliant C# blog, although his subjects are usually quite advanced.
+- The [C#/.NET Little Wonders & Little Pitfalls](http://geekswithblogs.net/BlackRabbitCoder/archive/2015/04/02/c.net-little-wonders-amp-little-pitfalls-the-complete-collection.aspx) posts by James Michael Hare contains some fantastic posts on less known C# features.
 
 ### Social media
-* [StackOverflow ](http://stackoverflow.com/questions/tagged/c%23) can be used to search for your problem and see if it has been answered already. You can also ask and answer questions.
-* [/r/csharp](https://www.reddit.com/r/csharp) is the C# subreddit.
-* [\@dotnet](https://twitter.com/DotNet) is the official .NET Twitter account.
-* [Gitter](https://gitter.im/exercism/xcsharp) is the C# Gitter room, go here to get support and ask questions related to the C# track.
+
+- [StackOverflow ](http://stackoverflow.com/questions/tagged/c%23) can be used to search for your problem and see if it has been answered already. You can also ask and answer questions.
```
Comment (jpreese) [inline docs/RESOURCES.md]: extra space in the tag

File: exercises/clock/README.md [hunk 2]
```diff
@@ -6,6 +6,20 @@ You should be able to add and subtract minutes to it.
 
 Two clocks that represent the same time should be equal to each other.
 
+## Hints
+This exercise requires you to implements a type-specific method for determining equality of instances.
+For more information, see [this page]
+(https://docs.microsoft.com/en-us/dotnet/core/api/System.IEquatable-1) .
```
Comment (jpreese) [inline exercises/clock/README.md]: space after period

File: exercises/clock/README.md [hunk 3]
```diff
@@ -6,6 +6,20 @@ You should be able to add and subtract minutes to it.
 
 Two clocks that represent the same time should be equal to each other.
 
+## Hints
+This exercise requires you to implements a type-specific method for determining equality of instances.
```
Comment (jpreese) [inline exercises/clock/README.md]: implement*

File: exercises/pig-latin/README.md [hunk 4]
```diff
@@ -7,16 +7,24 @@ confusing. It obeys a few simple rules (below), but when it's spoken
...
 to understand.
 
-- **Rule 1**: If a word begins with a vowel sound, add an "ay" sound to
-  the end of the word.
-- **Rule 2**: If a word begins with a consonant sound, move it to the
-  end of the word, and then add an "ay" sound to the end of the word.
+- **Rule 1**: If a word begins with a vowel sound, add an "ay" sound to the end of the word. Please note that "xr" and "yt" at the beginning of a word make vowel sounds (e.g. "xray" -> "xrayay", "yttria" -> "yttriaay").
+- **Rule 2**: If a word begins with a consonant sound, move it to the end of the word and then add an "ay" sound to the end of the word. Consonant sounds can be made up of multiple consonants, a.k.a. a consonant cluster (e.g. "chair" -> "airchay").
+- **Rule 3**: If a word starts with a consonant sound followed by "qu", move it to the end of the word, and then add an "ay" sound to the end of the word (e.g. "square" -> "aresquay").
+- **Rule 4**: If a word contains a "y" after a consonant cluster or as the second letter in a two letter word it makes a vowel sound (e.g. "rhythm" -> "ythmrhay", "my" -> "ymay").
```
Comment (jpreese) [inline exercises/pig-latin/README.md]: this sentence seems a little confusing.. "..cluster or as the second letter..."

File: exercises/scale-generator/README.md [hunk 5]
```diff
@@ -20,9 +20,9 @@ collection. They have seven pitches, and are called diatonic scales.
 The collection of notes in these scales is written with either sharps or
 flats, depending on the tonic. Here is a list of which are which:
 
-No Accidentals:
+No Sharps or Flats:
 C major
-A minor
+a minor
```
Comment (jpreese) [inline exercises/scale-generator/README.md]: shouldnt this be left as capital?

**Comment by ErikSchierboom** [comment]
@jpreese Thanks for the review! I've updated the PR (please squash before merging please).

**Comment by jpreese** [comment]
Looks like a test failed, rerunning build to double check.

**Comment by ErikSchierboom** [comment]
Thanks!

**Comment by jpreese** [comment]
🎉

**Pull request merged**